apps/src/sites/studio/pages/schoolInfo.js

```diff
@@ -293,6 +293,7 @@ window.SchoolInfoManager = function (existingOptions) {
   });
 
   $('#school-district-other').change(function () {
```

Comment:
Note that I did not make an analogous change below, at line 313 in the $('#school-other') section - that field seems to work properly as-is for some reason, so I didn't want to mess with it. I do wonder if there might be other forms where we make a similar mistake, though.